<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kttsjobmgr</class>
 <widget class="QWidget" name="kttsjobmgr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jobs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Speech Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="KPushButton" name="stopButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="cancelButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="whatsThis">
           <string>&lt;p&gt;Deletes the job.  If it is currently speaking, it stops speaking.  The next speakable job in the list begins speaking.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="pauseButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="whatsThis">
           <string>&lt;p&gt;Changes a job to Paused state.  If currently speaking, the job stops speaking. Paused jobs prevent jobs that follow them from speaking, so either click &lt;b&gt;Resume&lt;/b&gt; to make the job speakable, or click &lt;b&gt;Later&lt;/b&gt; to move it down in the list.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="resumeButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="whatsThis">
           <string>&lt;p&gt;Resumes a paused job or changes a Queued job to Waiting.  If the job is the top speakable job in the list, it begins speaking.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>Resume</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="KPushButton" name="speak_clipboard">
          <property name="whatsThis">
           <string>&lt;p&gt;Queues the current contents of the clipboard for speaking and sets its state to Waiting.  If the job is the topmost in the list, it begins speaking.  The job will be spoken by the topmost Talker in the &lt;b&gt;Talkers&lt;/b&gt; tab.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Speak Clipboard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="speak_file">
          <property name="whatsThis">
           <string>&lt;p&gt;Prompts you for a file name and queues the contents of the file for speaking.  You must click the &lt;b&gt;Resume&lt;/b&gt; button before the job will be speakable.  The job will be spoken by the topmost Talker in the &lt;b&gt;Talkers&lt;/b&gt; tab.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>Spea&amp;k File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="job_changetalker">
          <property name="whatsThis">
           <string>&lt;p&gt;Prompts you with a list of your configured Talkers from the &lt;b&gt;Talkers&lt;/b&gt; tab.  The job will be spoken using the selected Talker.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>Change Talker</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="TalkerWidget" name="talkerWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>TalkerWidget</class>
   <extends>QWidget</extends>
   <header>talkerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
